src/rules/use-tokens/use-tokens.js

```diff
@@ -59,44 +48,7 @@ const BORDER_PROPS = [
   'borderWidth',
 ];
 
-const SPACING_PROPS = [
```

Comment:
We don't track spacing now as the Backpack variables are no longer in JS and you can only set them via CSS